packages/opentelemetry/README.md

```diff
@@ -135,6 +145,35 @@ setupOpenTelemetry({ /* ... */ });
     </tr>
 </table>
 
+### Running in production
+
+To use this package in production you'll need a [Collector](https://opentelemetry.io/docs/collector/) that can receive traces over HTTP. This could be something you run (e.g. the [AWS Distro for OpenTelemetry](https://aws.amazon.com/otel/)) or a third-party service.
+
+Having traces collected centrally will give you a good view of how your production application is performing, allowing you to debug issues more effectively.
+
+OpenTelemetry can generate a huge amount of data which, depending on where you send it, can become very expensive. In production environments where you don't have control over the traffic volume of your app, you'll likely need to sample your traces. This package automatically samples traces ([at 5% by default](#optionstracingsamplepercentage)).
+
+### Running locally
+
+If you want to debug specific performance issues then setting up a local Collector can help you. You shouldn't be sending traces in local development to your production backend as this could make it harder to debug real production issues. You probably also don't want to sample traces in local development – you'll want to collect all traffic because the volume will be much lower.
+
+#### Running a backend
+
+To view traces locally, you'll need a backend for them to be sent to. In this example we'll be using [Jaeger](https://www.jaegertracing.io/) via [Docker](https://www.docker.com/). You'll need Docker (or a compatible [alternative](https://podman.io/)) to be set up first.
+
+[Jaeger maintains a useful guide for this](https://www.jaegertracing.io/docs/1.53/getting-started/#all-in-one).
+
+#### Sending traces to your local backend
+
+Once your backend is running you'll need to make some configuration changes.
+
+You'll need to set the [tracing endpoint](#optionstracingendpoint) to use Jaeger's tracing endpoint on port `4318` ([OTLP/HTTP](https://opentelemetry.io/docs/specs/otlp/#otlphttp)). E.g. `http://localhost:4318/v1/traces`.
+
+You'll also need to disable sampling by [configuring it](#optionstracingsamplepercentage) to `100`.
+
```

Comment:
suggestion: would it be worth adding this as an example command (before line 174) to the README?

```suggestion
OPENTELEMETRY_TRACING_ENDPOINT=http://localhost:4318/v1/traces OPENTELEMETRY_TRACING_SAMPLE_PERCENTAGE=100 npm start
```

Reply:
Good shout 👍 I've updated this